events.config.yaml

```
@@ -10,16 +10,18 @@ event_providers:
       - event_code: com.3rdparty.events.test.Event1
         label: Test event 1
         description: Test event 1 for 3rd party system
-  - label: Commerce events provider
+  - label: Commerce events provider app one
```

Before merging, I'm removing `app one` from the label. This is just the user-friendly name of the provider and later on a postfix by script; it doesn't need to have the app name.
```suggestion
  - label: Commerce events provider
```